cygnus-ngsi/README.md

```diff
@@ -284,6 +285,7 @@ Detailed information regarding cygus-ngsi can be found in the [Installation and
   <tr><td>Raw-snapsot analysis</td><td>1.6.0</td></tr>
   <tr><td>NGSIOrionSink</td><td>First implementation</td><td>1.10.0</td></tr>
   <tr><td>NGSIElasticsearchSink</td><td>First implementation</td><td>1.15.0</td></tr>
+  <tr><td>NGSIArcGisSink</td><td>First implementation</td><td>1.15.0</td></tr>
```

Comment:
Cygnus 1.15.0 window has passed (it will be released today). Thus, please change this to 1.16.0.

Reply:
Done